source/wp-content/themes/wporg-documentation-2022/functions.php

```diff
@@ -46,11 +46,69 @@ function set_site_breadcrumbs( $breadcrumbs ) {
 	if ( is_front_page() ) {
 		return array();
 	}
-
 	$breadcrumbs[0]['title'] = __( 'Documentation', 'wporg-docs' );
```

Comment:
Is this overwriting the original first item? Is that the intention?

Reply:
Maybe it's the JS dev in me but I feel like this function should avoid mutation of `$breadcrumbs` and return `$newBreadcrumbs`

Reply:
Maybe I need to let go of that though 😂

Reply:
> Is this overwriting the original first item?

Yes, the first item's title defaults to the site title, but in this site's case the site title is "WordPress Forums". So we need to override it, since we're not updating the whole site right now.

Reply:
> It's definitely the JS dev in me, but I feel like this function should avoid mutation of $breadcrumbs and return $newBreadcrumbs

I get it, but that's really not an issue in PHP — also, it would be `$new_breadcrumbs` 😄 

If you think that would be more understandable, I'm happy to update it, though.

Reply:
Maybe just a comment to explain what's happening with the rename of the first

Reply:
Done in https://github.com/WordPress/wporg-documentation-2022/pull/19/commits/189a734907a29efe269b88aeb6f00774f6e9e45c